### PR TITLE
SDI-333 Changes to event handling to improve latency

### PR DESCRIPTION
**Pull request by steverendell**
Switch to use new service (item 6 in ticket)

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/JMSReceiverTest.kt [hunk 1]
```diff
@@ -48,15 +55,66 @@ class JMSReceiverTest {
...
+      this.jmsTimestamp = START - ONE_SECOND
+    }
+
+    service.onMessage(message)
+
+    verifyNoInteractions(eventsEmitter)
+  }
+
+  @Test
+  fun `will only log message if after JMS timestamp range`() {
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/JMSReceiverTest.kt:104]: should this be named `will only send message ...` ?

**Review by andymarke — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/JMSReceiverTest.kt (hunk 1 above)
Comment (steverendell) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/JMSReceiverTest.kt:104]: its after the end of the range so it is only logging , not sending

**Review by andymarke — APPROVED** [review]
lgtm

**Pull request merged**